### PR TITLE
Add target environment variable

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: gatling-job.groovy [hunk 1]
```diff
@@ -9,7 +9,7 @@ job('Gatling job') {
         }
     }
     steps {
-        sbt('sbt', '$SBT_ACTION', '-Dsbt.log.noformat=true') 
+            sbt('sbt', '$SBT_ACTION', '-Dsbt.log.noformat=true -DTARGET_HOSTNAME=james-charge-01')
```
Comment (aduprat) [inline gatling-job.groovy]: Why did you change the indent?

File: gatling-job.groovy [hunk 2]
```diff
@@ -1,15 +1,16 @@
 job('Gatling job') {
     parameters {
         stringParam('SBT_ACTION', '', 'sbt action, ex.: gatling:testOnly com.linagora.gatling.imap.scenario.ImapSimpleScenario')
+        stringParam('GITHUB', '', 'Github repository used, ex.: linagora/imap-gatling')
```
Comment (aduprat) [inline gatling-job.groovy:4]: s/imap-gatling/gatling-imap

Comment (blackheaven) [reply gatling-job.groovy:4]: They can choose

**Review by aduprat — APPROVED** [review]

File: DockerfileSlave [hunk 3]
```diff
@@ -10,8 +10,11 @@ RUN apt-get update \
     && echo deb https://apt.dockerproject.org/repo debian-jessie main >> /etc/apt/sources.list.d/docker.list \
     && apt-get update \
     && apt-get install -y docker-engine \
+    && apt-get install -y btrfs-tools \
```
Comment (rouazana) [inline DockerfileSlave:13]: the commit 7959a23e64b00d2ace830d44583dbc895141ddd5 is misnamed, it does not only "suffix jenkins file with groovy"

**Review by chibenwa — APPROVED** [review]

**Review by aduprat — APPROVED** [review]

**Review by rouazana — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]
